gdnative-core/src/export/property/hint.rs

```diff
@@ -136,13 +146,46 @@ impl EnumHint {
         }
 
         for rest in iter {
-            write!(s, ",{rest}").unwrap();
+            write!(s, ",").unwrap();
+            write!(s, "{rest}").unwrap();
         }
```

Comment:
Now that there's no longer a `write_item` function necessary, this can be reverted completely to what it was.